pixi/src/ui/page-experience.hbs

```diff
@@ -2,3 +2,4 @@
 {{#htmlWebpackPlugin.files.js }}
 {% set src = '{{ . }}' %}
 {{/htmlWebpackPlugin.files.js }}
+{% set cspHash = '{{ cspHash }}' %}
```

Comment:
would  there ever a case for multiple scripts on the page?

Reply:
In this context yes, only one hash, as Pixi's webpack config will only ever emit one bundle.

Reply:
... and yes, rendering `src` in a loop then is confusing 😅